src/main/java/core/basesyntax/AreaCalc.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface AreaCalc {
+    double getArea();
+
```

Comment:
remove empty line

Reply:
I don't how you were trying to delete this empty line, but you definitely can do this, next time if you have problems with  implementation of mentor's comments write to your private channel questions

Reply:
sorry, I thought you meant the empty line between import and class